manifests/1.2.0/opensearch-dashboards-1.2.0.yml

```diff
@@ -16,4 +16,19 @@ components:
 - name: securityDashboards
   repository: https://github.com/opensearch-project/security-dashboards-plugin.git
   ref: "main"
+- name: queryWorkbenchDashboards
+  repository: https://github.com/opensearch-project/sql.git
+  working_directory: workbench
+  ref: "1.2"
+- name: reportsDashboards
+  repository: https://github.com/opensearch-project/dashboards-reports.git
+  working_directory: dashboards-reports
+  ref: "1.2"
+- name: traceAnalyticsDashboards
+  repository: https://github.com/opensearch-project/trace-analytics.git
+  ref: "1.2"
+- name: ganttChartDashboards
+  repository: https://github.com/opensearch-project/dashboards-visualizations
+  working_directory: gantt-chart
+  ref: "1.2"
```

Comment:
This was is all good! `1.2` branch exists and plugin is bumped! If anything it might be easiest to pull this one into another PR so that it's not blocked waiting on the other PRs.